src/esp/gfx/ssao/nvpro_core/nvh/cameramanipulator.cpp

```diff
@@ -530,10 +530,10 @@ void CameraManipulator::fit(const nvmath::vec3f& boxMin,
       {
         // Keep the largest offset to see that vertex
         offset =
-            std::max(fabs(vct.y) / tan(nv_to_rad * yfov * 0.5f) + fabs(vct.z),
+            std::max(double(fabs(vct.y)) / tan(nv_to_rad * yfov * 0.5f) + fabs(vct.z),
```

Comment:
Wouldn't it just be better to do:
```suggestion
            std::max((fabs(vct.y)) / std::tan(nv_to_rad * yfov * 0.5f) + std::fabs(vct.z),
```